index.js

```diff
@@ -13,11 +13,15 @@ async function run() {
   try {
     const platform = getVirtualEnvironmentName()
     const ruby = await getRubyEngineAndVersion(core.getInput('ruby-version'))
+    const reJRuby = /^jruby/i
 
     let rubyPrefix
     if (platform === 'windows-latest') {
       rubyPrefix = await windows.downloadExtractAndSetPATH(ruby)
     } else {
+      if (reJRuby.test(ruby)) {
```

Comment:
Could you change it to `if (ruby.startsWith('jruby')) {` ?
That's good enough, `getRubyEngineAndVersion` ensures the engine name is lower case.

Reply:
Done